frontend/src/Context/PublisherProvider/usePublisher/usePublisher.tsx

```diff
@@ -89,7 +122,15 @@ const usePublisher = (): PublisherContextType => {
     publisherRef.current = null;
   };
 
-  const handleAccessDenied = () => {
+  const handleAccessDenied = (event: AccessDeniedEvent) => {
+    // We check the first word of the message to see if the microphone or camera was denied access.
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith 
?

Reply:
or includes to be more lenient